scripts/generatePagesFromConsolidatedJSON.py

```diff
@@ -784,7 +784,8 @@ def write_index_step2(findex, data):
               </ul>
 
                 <p style="text-align:left">For the last two cases, you
-                can either submit a proper JSON file as a <a
+                can either submit a proper <a
+                href="https://github.com/GraphicsReplicability/replicability.graphics/blob/master/template.json">JSON file</a> as a <a
```

Comment:
The template is given slightly below.
Do you want to keep the two links?

Reply:
yes.... it is mentionned way too late right now (upon reading the first mention of JSON, one wonders where it is), and it doesn't cost anything to put it twice

Reply:
ok